src/components/Text/types.ts

```diff
@@ -16,5 +16,5 @@ export enum TextVariants {
   ctaSm = 'ctaSm',
   email = 'email',
   label = 'label',
-  textBadge = 'textBadge',
+  badge = 'badge',
```

Comment:
Needs to be updated in story control dropdown

Reply:
this looks right to me.  Sometimes you have to close and reopen storybook:
![image](https://github.com/PhillipsAuctionHouse/seldon/assets/6663002/2f717f63-7964-4b94-9524-bff82e86acd0)

Reply:
Looks good now